src/core/src/op/is_nan.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/is_nan.hpp"
+
+#include "itt.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
```

Comment:
Is this include required?

Reply:
Yes, it is required.

Reply:
Don's see usage of `host_tensor` here, am I missing something?

Reply:
Sorry, I misunderstood you. `"ngraph/runtime/host_tesnor.hpp"` is not required and I will remove it.